task1/src/GenreToggle.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+export const GenreToggle = () => {
+    return (
```

Comment:
can be without return.

U can leave as is)

Reply:
fixed